README.md

```diff
@@ -0,0 +1,13 @@
+# propagator
+
+FD simulation for 3D elastic wave equation.
+
+running the python script codegen/grid3d.py (this script is the same as the notebook grid3d.ipynb), this will generate test3d.cpp. This can be compiled manually with g++ -fopenmp for instance, or build the make file according to the .travis scrip.
```

Comment:
The typo is still there: https://raw.githubusercontent.com/opesci/propagator/codegen/README.md